src/java/org/apache/cassandra/sensors/SensorsRegistry.java

```diff
@@ -95,6 +99,24 @@ public class SensorsRegistry implements SchemaChangeListener
     private SensorsRegistry()
     {
         Schema.instance.registerListener(this);
+        MessagingService.instance().outboundSink.add(this::trackOutboundMessages);
+    }
+
+    /**
+     * Tracks outbound messages size and count in Sensors Regsitry
+     */
+    private boolean trackOutboundMessages(Message<?> message, InetAddressAndPort ignored)
```

Comment:
Adding internode tracking responsibility to the registry doesn't look right to me: what about moving this code to `MessagingService#listen()` and rewrite it slightly to use the "standard" sensors API? It would probably be slightly more verbose, but overall cleaner IMO. WDYT?